mod/admin.php

```diff
@@ -100,6 +105,8 @@ function __call_theme_admin_post(&$a, $theme) {
 }
 
 /**
+ * This function generates the content for the admin panel.
```

Comment:
I think it should be the motivation to expand it. There are also other functions without docs left in admin.php but I could not come up with any for those.